packages/eslint-config-terra/README.md

```diff
@@ -0,0 +1,66 @@
+# Eslint Config Terra
+
+[![NPM version](https://badgen.net/npm/v/@carbon/eslint-config-terra)](https://www.npmjs.org/package/@carbon/eslint-config-terra)
+[![Build Status](https://badgen.net/travis/cerner/terra-toolkit)](https://travis-ci.com/cerner/terra-toolkit)
+
+Terra's sharable ESLint configuration for their UI library and build tools.
+
+Features:
+
+- extends the  [eslint-config-airbnb](https://github.com/airbnb/javascript/tree/master/packages/eslint-config-airbnb), which exports ESLint rules, including rules for ECMAScript 6+ and React.
+- checks for proper ReactJS Hooks usage via the [eslint-plugin-react-hooks](https://reactjs.org/docs/hooks-rules.html).
+- checks syntax for targeted browsers support via the [eslint-plugin-compat](https://github.com/amilajack/eslint-plugin-compat). It is recommended to use Terra's targeted browsers which are specified by the [browserslist-config-terra](https://github.com/cerner/terra-toolkit/tree/master/packages/browserslist-config-terra) module.
+- defines the browser and jest environment
+- defines enzyme globals for `shallow`, `render` and `mount` for jest testing
+
+## What is Eslist
+
+[ESlint](https://eslint.org/) is a pluggable linting utility for JavaScript and JSX.
+
+## Installation
+
+Install the module
+
+```shell
+npm install eslint --save-dev
+npm install eslint-config-terra --save-dev
+```
+
+## Usage
+
+First, include the configuration defined by `eslint-config-terra` via the  [extends](https://eslint.org/docs/user-guide/configuring#extending-configuration-files) property.
+
+Then, [define the browsers](https://github.com/amilajack/eslint-plugin-compat#targeting-browsers) for the `eslint-plugin-compat` plugin to target by add the browsers list configuration in your package.json through the `browserslist` key.
+
+### package.json
+
+```json
+{
+  "browserslist": [
+    "extends browserslist-config-terra"
+  ],
+  "eslintConfig": {
+    "extends": "terra"
```

Comment:
with moving to scoped packages, can we still use like this?

Reply:
```suggestion
    "extends": "@cerner/terra"
```